src/bin/wapm.rs

```diff
@@ -27,23 +32,26 @@ enum Command {
     /// Install a package
     Install(commands::InstallOpt),
 
+    #[cfg(feature = "full")]
     #[structopt(name = "publish")]
     /// Publish a package
     Publish(commands::PublishOpt),
 
+    #[cfg(feature = "full")]
     #[structopt(
         name = "run",
         settings = &[AppSettings::TrailingVarArg, AppSettings::AllowLeadingHyphen],
     )]
     /// Run a command from the package or one of the dependencies
     Run(commands::RunOpt),
 
+    #[cfg(feature = "full")]
```

Comment:
Can we re-enable this one as well?

Reply:
Run command will be super cool

Reply:
I have now implemented this ... including the pre-dir support :-)

```
wapm install quickjs
wapm run quickjs --dir /
```

See it in action here:
https://tokera.sh/?init=wapm%20install%20quickjs%20%26%26%20wapm%20run%20quickjs%20--dir%20%2F